src/pdf/ShoppingList/shoppingListPdfDataProps.ts

```diff
@@ -89,10 +91,11 @@ export const prepareItemsListForHousehold = async (
         if (listItemError) {
             return { data: null, error: listItemError };
         }
-        itemsList.push({
-            description: row.item_name,
-            ...listItemData,
-        });
+        row.list_type === listType &&
+            itemsList.push({
+                description: row.item_name,
+                ...listItemData,
```

Comment:
this pattern for type checking seems a little strange in a .ts file but seems to work so no real issue. maybe a basic if statement would be better?